src/figuresClasses.ts

```diff
@@ -1,19 +1,109 @@
-export interface Figure {
+enum EnumShape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
+
+enum EnumColor {
```

Comment:
pls refactor all enums according to [enums naming convention](https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/typescript/data-types/enums.html#:~:text=Naming%20convention,enum%20tend%20to%20be%20titlecase)